pkg/controller/etcdproxy/etcdproxy_controller.go

```diff
@@ -273,6 +285,17 @@ func (c *EtcdProxyController) syncHandler(key string) error {
 		return err
 	}
 
+	// Deploy Server Etcd Proxy certificates.
+	if err = c.createEtcdProxyServerCACerts(etcdstorage); err != nil {
```

Comment:
TODO: Consider name change. Potentially `createEtcdProxyCACerts` could be a better name.